src/graphics/program-lib/programs/standard.js

```diff
@@ -334,29 +349,29 @@ const standard = {
             // albedo
             decl.append("vec3 dAlbedo;");
             if (options.diffuseDetail) {
-                code.append(this._addMap("diffuseDetail", "diffuseDetailMapPS", options, litShader.chunks));
+                code.append(this._addMap("diffuseDetail", "diffuseDetailMapPS", options, litShader.chunks, textureMapping, options.diffuseEncoding));
```

Comment:
diffuseDetail doesn't necessarily have the same encoding as diffuse

Reply:
Ah! Good catch, will fix that!

Reply:
https://github.com/playcanvas/developer.playcanvas.com/pull/439

Here's the PR for the chunk migrations.